src/components/Map.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useRef, useEffect, useMemo } from 'react';
+import mapboxgl from 'mapbox-gl';
+
+const ACCESS_TOKEN = 'pk.eyJ1Ijoibmljb2xhc21hcnRpbmV6MDUxMCIsImEiOiJja3cwbW84Z2VkaHhxMnZwZzIyODg5Nzk4In0.3WRMNapO4tRArFLEoenJKQ'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/FeNixCrew/MiEntradita-frontend/pullRequest?prid=8421134)
```suggestion
const ACCESS_TOKEN = "pk.eyJ1Ijoibmljb2xhc21hcnRpbmV6MDUxMCIsImEiOiJja3cwbW84Z2VkaHhxMnZwZzIyODg5Nzk4In0.3WRMNapO4tRArFLEoenJKQ"
```